compiler/luci-interpreter/src/kernels/Pad.test.cpp

```diff
@@ -26,6 +26,34 @@ namespace
 
 using namespace testing;
 
+// for quantized Add, the error shouldn't exceed step
+float GetTolerance(int min, int max) { return (max - min) / 255.0; }
```

Comment:
min, max should be float.

please see https://github.com/Samsung/ONE/pull/2800/files#r448751543.

Reply:
Oh sorry! I will fix it!